src/groups/semidirect_product_group.jl

```diff
@@ -135,41 +135,6 @@ function _compose!(G::SemidirectProductGroup, x, p, q)
     return x
 end
 
-@doc raw"""
```

Comment:
Are these docs kept somewhere?
Or phrased differently: It is maybe useful to keep these attached to a new function or at least to the signature here?

Reply:
I think that this docstring is now obsolete.

Reply:
Ok.